lib/NVS/src/Namespace.cpp

```diff
@@ -20,18 +20,21 @@ Namespace::~Namespace() {
     }
 }
 
-esp_err_t Namespace::open(const char* name, OpenMode mode) {
+esp_err_t Namespace::open(const char* name, nvs_open_mode mode) {
     // A namespace is already opened in this instance.
     if (_isOpened) {
         return ESP_FAIL;
     }
-
+    ESP_LOGI(__func__,"-------------------------------vamos a abrie el namespacen\n");
```

Comment:
Remove the hypens from `ESP_LOGx` log functions, log messages should be concise, also, would you consider writting them in english? So we don't mix up languages inside the code